ding_availability.module

```diff
@@ -210,7 +221,7 @@ function ding_availability_holdings($provider_ids) {
  *
  * @return array
  */
-function ding_availability_holdings_row($holding) {
+function ding_availability_holdings_row($holding, $ordered_count) {
```

Comment:
Could you rename `$ordered_count` to `$show_ordered_count` (or something like that) and update the docblock?

As far as I can tell the current value passed to the method is the actual total count but the method actually only wants a boolean flag to determine whether to show the ordered count for the individual row or not.